CONTRIBUTING.md

```diff
@@ -1,100 +1,239 @@
 
-## Getting started
+Explore this guide to find out how to get started or join our [Slack community](https://slack.sanity.io) if you have any questions.
 
-Before contributing, please read our [code of conduct](https://github.com/sanity-io/sanity/blob/current/CODE_OF_CONDUCT.md).
+We would like to emphasize that writing code is not the only way you can contribute to Sanity. Submitting security issues, feature requests, and bug reports, as well as writing blog posts or tutorials, answering questions about Sanity, or organizing community events around Sanity are all considered meaningful, important, and much appreciated contributions to Sanity.
 
-Then make sure you have _Node.js version 10 or newer_ and _npm version 6 or newer_.
+## Code of Conduct
+
+Before contributing, please read our [code of conduct](https://github.com/sanity-io/sanity/blob/master/CODE_OF_CONDUCT.md). We require everyone who wants to contribute to Sanity to have read and understood our Code of Conduct. Violations of our Code of Conduct are taken seriously and will be dealt with accordingly.
+
+## What is the Sanity Studio?
+
+The [Sanity Studio](https://www.sanity.io/docs/sanity-studio) is an open-source editing environment built with React.
+
+## Branches
+
+- `current`: This contains all the features and fixes included in the latest official release.
+- `next`: This includes everything scheduled for the next, upcoming release.
+
+## Accepted contributions
+
+- Bug reports
+- Feature requests
+- Bug fixes
+- Smaller improvements/enhancements of existing features
+- New features
+- Documentation improvements
+
+New features are unlikely to be accepted and are better developed and published as plugins that extend the existing functionality of the Sanity Studio. See [New features](#new-features) for more info.
+
+### Security issues
+
+If you find a security vulnerability, do **NOT** open an issue.
+
+Email `security@sanity.io` instead.
+
+Any security issues should be submitted directly to `security@sanity.io`. In order to determine whether you are dealing with a security issue, ask yourself these two questions:
+
+- Can I access something that's not mine, or something I shouldn't have access to?
+- Can I disable something for other people?
+
+If the answer to either of those two questions are "yes", then you're probably dealing with a security issue. Note that even if you answer "no" to both questions, you may still be dealing with a security issue, so if you're unsure, just email us at `security@sanity.io`.
+
+### Bug reports
+
+To submit a bug report, check if the issue has already been reported, and if not, please create a new issue and use the provided template. Make sure your report answers these five questions:
```

Comment:
Should we just point to the template and let that tell what needs to be answered?